src/lib/components/get-demo/calendly-embed.svelte

```diff
@@ -0,0 +1,23 @@
+<style lang="postcss">
+  div :global(iframe) {
+    @apply rounded-2xl shadow-normal;
+  }
+</style>
+
+<svelte:head>
+  <link
+    href="https://calendly.com/assets/external/widget.css"
+    rel="stylesheet"
+  />
+  <script
+    async
+    src="https://calendly.com/assets/external/widget.js"
+    type="text/javascript">
+  </script>
+</svelte:head>
+
+<div
+  class="calendly-inline-widget"
+  data-url="https://calendly.com/d/d47-7n4-c63?primary_color=ffae33"
+  style="min-width:250px;height:650px;"
```

Comment:
We can change to following, to remove the scroll bars:

| Before | After  |
|--------|---|
| ![image](https://user-images.githubusercontent.com/55068936/190449777-530d8b9f-789c-4264-8dce-e71c7cbb69c8.png)        | ![image](https://user-images.githubusercontent.com/55068936/190449821-23134f4b-19c1-4788-ad29-8fcb1ee2d73b.png)  |


```suggestion
  style="min-width:250px;height:726px;"
```